src/System.ObjectModel/tests/System.ObjectModel.Tests.csproj

```diff
@@ -24,6 +24,7 @@
     </Compile>
     <Compile Include="ComponentModel\INotifyPropertyChangingTests.cs" />
     <Compile Include="ComponentModel\PropertyChangingEventArgsTests.cs" />
+    <Compile Include="System\ComponentModel\TypeConverterAttributeTests.cs" />
```

Comment:
Nit, we generally sort, it makes merges easier sometimes 😺

Reply:
Actually this is correct - first you order the folder (i.e. `ComponentModel` before `System`) then the file (i.e. `System\ComponentModel\A.cs` before `System\ComponentModel\B.cs`), then files without a folder

The problem here is that the files have a bad/wrong/weird folder layout that wilk be fixed in future PRs